### ISSUE TITLE
PyTorch Tutorial - Gradient Descent | Hex.*

### ISSUE THREAD

**Issue by HexuanWeng** [HexuanWeng/hexuanweng.github.io#42]
https://hexuanweng.github.io/2022/07/14/PyTorch-P3/ 

 Lecture3 -- Bilibili: PyTorch Tutorial; Lecturer: Hongpu Liu
